iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -93,19 +94,13 @@ class storable_function<Capacity, signature<ReturnType, Args...>>
     /// @param args arguments to invoke the stored function with
     /// @return return value of the stored function
     ///
-    /// @note 1) Invoking the function if there is no stored function (i.e. operator bool returns false)
```

Comment:
This was incorrect. 
1. It will now not terminate anymore since it cannot be null by design (except after move, but this is illegal).

2. It now has noexcept (already had it) but actually should not since the callable we store can happily throw. This is an issue in all our template code IMO.